product_template_multi_link/tests/test_product_template_linker.py

```diff
@@ -84,7 +84,7 @@ def test_wizard_link_duplicate1(self):
         self._test_link_created(links, link_type)
         # Ensure no duplicates
         link = self.product1.product_template_link_ids.filtered(
-            lambda l: l.right_product_tmpl_id == self.product2
+            lambda r: r.right_product_tmpl_id == self.product2
```

Comment:
@David-Luis-Mora Was this change made for any specific reason?

Reply:
Yes, because when I made the pre-commit it was giving me an error, so I had to make that change.